spec/controllers/ipeds_hds_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe IpedsHdsController, type: :controller do
+end
```

Comment:
Let's not add any of these controller spec stubs until you've written up the controllers. No point merging these changes in as they provide no new functionality.